README.md

```diff
@@ -1,14 +1,76 @@
 # Mines ACM Student Chapter Website
 
+[acm.mines.edu](https://acm.mines.edu/)
+
+This is the website for the Colorado School of Mines' local chapter for the [Association for Computing Machinery](http://www.acm.org/).
```

Comment:
Please truncate all lines to a maximum of 80 characters. (Unless it's a CLI example, like below.)